core/core-impl/src/main/java/ai/timefold/solver/core/api/solver/SolverJob.java

```diff
@@ -95,6 +95,14 @@ public interface SolverJob<Solution_, ProblemId_> {
      */
     long getScoreCalculationCount();
 
+    /**
+     * Return the {@link ProblemStatistics} for the {@link PlanningSolution problem} submitted to the
+     * {@link SolverManager}.
```

Comment:
I'm wondering if we should describe the volatility of this value.
It may be something else at solver start, during solver run, and at the end. (`ProblemChange` again.)